aspnetcore/mvc/views/tag-helpers/th-components/samples/RazorPagesSample/Pages/Shared/_CookieConsentPartial.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@using Microsoft.AspNetCore.Http.Features
+
+@{
+    var consentFeature = Context.Features.Get<ITrackingConsentFeature>();
+    var showBanner = !consentFeature?.CanTrack ?? false;
```

Comment:
This is interesting to me in passing ... `!(null)` ... I guess it just evals to `null`? ... is that right?